src/components/dashboard/ReportModal.tsx

```
@@ -0,0 +1,243 @@
+        </div>
+
+        <div className="p-6">
+          <form
+            onSubmit={(e) => {
+              e.preventDefault();
+              handleDownload();
+            }}
+          >
+            <div className="grid grid-cols-1 md:grid-cols-2 gap-6 mb-6">
+              <div>
+                <label className="block text-sm font-medium text-gray-700 mb-1">From Date</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)